services_app/src/main/java/org/opendatakit/services/MainActivity.java

```diff
@@ -143,38 +336,60 @@ public boolean onOptionsItemSelected(MenuItem item) {
       return true;
     }
 
-    if (id == R.id.action_resolve_conflict) {
+    if (id == R.id.action_settings) {
+
+      Intent intent = new Intent(this, AppPropertiesActivity.class);
+      intent.putExtra(IntentConsts.INTENT_KEY_APP_NAME, getAppName());
+      startActivityForResult(intent, SETTINGS_ACTIVITY_RESULT_CODE);
+      return true;
+    }
+
+    return false;
+  }
+
+  // Action on Clicking the Drawer Menu Item
+  private boolean onDrawerMenuItemClicked(MenuItem item){
+
+    drawerLayout.closeDrawer(GravityCompat.START);
+
+    if(item.getItemId()==R.id.drawer_resolve_conflict){
```

Comment:
Use `switch case` whenever your if/ifelse exceeds 3

Reply:
The `if/else if` is fine here. Resource IDs will be non-final in an upcoming version of AGP. There is a lint warning for this:
 

> Resource IDs will be non-final in Android Gradle Plugin version 5.0, avoid using them in switch case statements
> Avoid the usage of resource IDs where constant expressions are required.
>
> A future version of the Android Gradle Plugin will generate R classes with non-constant IDs in order to improve the performance of incremental compilation.
> To suppress this error, use the issue id "NonConstantResourceId" as explained in the Suppressing Warnings and Errors section.